delphi/MarkdownUITestForm.pas

```diff
@@ -47,10 +60,22 @@ procedure TMarkDownUITest.Button1Click(Sender: TObject);
 procedure TMarkDownUITest.Button2Click(Sender: TObject);
 var
   DaringFireball : TMArkdownProcessor;
+  Doc: IHTMLDocument2;
+  psa: PSafeArray;
+  v: variant;
+  s: String;
 begin
   DaringFireball := TMarkdownProcessor.CreateDialect(mdDaringFireball);
   try
-    memo2.Text := DaringFireball.process(memo1.text);
+    s := DaringFireball.process(memo1.text);
+    memo2.Text := s;
+    if WebBrowser1.Document = nil then
+	    WebBrowser1.Navigate('about:blank');
+    v := VarArrayCreate([0, 0], varVariant);
+    v[0] := s;
+    psa := PSafeArray(TVarData(v).VArray);
+    Doc := WebBrowser1.Document as IHTMLDocument2;
+    Doc.write(psa);

```

Comment:
Duplicated code, but implementing new function will take more lines of code, then current version.